app/jobs/regular/remove_banner.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Jobs
+
+  class RemoveBanner < ::Jobs::Base
+
+    def execute(args)
+      topic_id = args[:topic_id]
+
+      raise Discourse::InvalidParameters.new(:topic_id) unless topic_id.present?
+
+      topic = Topic.find_by(id: topic_id)
+      topic.remove_banner!(Discourse.system_user) if topic.present?
```

Comment:
Could also be a two-liner.

```suggestion
      raise Discourse::InvalidParameters.new(:topic_id) unless args[:topic_id].present?
      Topic.find_by(id: args[:topic_id])&.remove_banner!(Discourse.system_user)
```